appveyor.yml

```diff
@@ -2,10 +2,6 @@ version: 6.0.0-alpha.18-{build}
 
 image: Visual Studio 2015
 
-cache:
-  - '%LocalAppData%\Microsoft\dotnet'
-  - '%HOMEDRIVE%%HOMEPATH%\.nuget\packages'
```

Comment:
@TravisEz13 talked to me offline and it turns out it's quite easy to invalidate the cache, I will do that.